pkg/i2gw/notifications/notifications.go

```diff
@@ -73,9 +73,10 @@ func (na *NotificationAggregator) CreateNotificationTables() []*tablewriter.Tabl
 			t.Append(row)
 		}
 
-		tables = append(tables, t)
+		tableString.WriteString(fmt.Sprintf("Notifications from %v\n", provider))
+		t.Render()
 	}
-	return tables
+	return tableString
```

Comment:
correct me if I am wrong, but it looks like you hold all the tables in one string `tableString`. Is there a reason you dont do as we said to have a list of `tableStrings` ?

Reply:
Oh i think i missed that, I ll make the change. Do you think it would be better to have a map of provider to string instead of a list.

Reply:
I dont think thats necessary now. If you can think of any use of it *now* - do it.

Reply:
It will be easier to parse and use in my opinion thats all. Plus we get a reference to the provider generating the table this way. We can easy change to a list implementation later is we feel it is unnecessary